pep/app/pods/components/page/nav/template.hbs

```diff
@@ -32,7 +32,7 @@
         {{! TODO project reqs state that there should be a "Read" nav item         }}
         {{! but where does it go/show? Seems like it would be redundant w/"Browse" }}
         <li class="nav-item">
-            <LinkTo @route="read" class="nav-link">
+            <LinkTo @route="read" @disabled={{this.readDisabled}} class="nav-link">
```

Comment:
maybe not doable for this initial merge, but I think we should at least consider having this link go to some sort of landing/"empty state" page (e.g. w/some content that states "Your last viewed document will be accessible here...", similar to the old "empty" search page layout maybe), instead of being outright disabled.

This is just a highly unusual pattern from a pure UX perspective. As a new user coming into the site fresh, I see a top-level "Read" link front and center in the main navigation, but its disabled...why? The user has no context or idea what that link should do.

cc @adistasio